### PR TITLE
Add commands in executeCommands

### PR DESCRIPTION
**Pull request by jLereback**
Add tests where needed

### CODE REVIEW

**Review by RobertMili — APPROVED** [review]

**Review by RobertMili — APPROVED** [review]

**Comment by Henrik-Bengtsson** [comment]
In HazeDatabaseTest the last test needs a rename. It's called only test now.

File: src/main/java/org/fungover/haze/Main.java [hunk 1]
```diff
@@ -78,15 +78,17 @@ public static String executeCommand(HazeDatabase hazeDatabase, List<String> inpu
         String command = inputList.get(0).toUpperCase();
 
         return switch (command) {
+            case "SET" -> hazeDatabase.set(inputList);
+            case "GET" -> hazeDatabase.get(inputList.get(1));
```
Comment (Ahlberg-iths) [inline src/main/java/org/fungover/haze/Main.java:82]:
```suggestion
            case "GET" -> hazeDatabase.get(inputList);
```

When GET is called without an argument we will get an index out of bounds exception, so maybe send in the whole list and then edit the set method to adjust for that.

**Review by Ahlberg-iths — CHANGES_REQUESTED** [review]

File: src/test/java/org/fungover/haze/HazeDatabaseTest.java [hunk 2]
```diff
@@ -93,41 +96,51 @@ void testPingResponseShouldBeSameAsValue() {
...
     }
+
+    @ParameterizedTest
+    @CsvSource({
+            "SET, key",
+            "SETNX, key"
+    })
+    void callingExecuteCommandWithWrongNumberOfArgumentsResultsInErrorMessage(String command, String message) {
+        assertThat(Main.executeCommand(testDatabase, List.of(command, message)))
+                .isEqualTo("-ERR wrong number of arguments for command\r\n");
```
Comment (Ahlberg-iths) [inline src/test/java/org/fungover/haze/HazeDatabaseTest.java:144]: Looks great, but maybe move it to MainTest instead

**Review by Ahlberg-iths — CHANGES_REQUESTED** [review]

**Review by Ahlberg-iths — APPROVED** [review]
Great job!

**Review by safstromo — APPROVED** [review]
:1st_place_medal:

**Pull request merged**